packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -0,0 +1,208 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Widget _boilerplate(VoidCallback onButtonPressed) {
+    return Directionality(
+      textDirection: TextDirection.ltr,
+      child: Stack(
+        children: <Widget>[
+          FlatButton(
+            child: const Text('TapHere'),
+            onPressed: onButtonPressed,
+          ),
+          DraggableScrollableSheet(
+              alignment: Alignment.bottomCenter,
+              maxChildSize: 1.0,
+              minChildSize: .25,
+              builder:
+                  (BuildContext context, ScrollController scrollController) {
+                return Container(
+                  color: const Color(0xFFABCDEF),
+                  child: ListView.builder(
+                    controller: scrollController,
+                    itemCount: 100,
+                    itemBuilder: (BuildContext context, int index) => Text(
```

Comment:
Can be a one liner:  `itemBuilder: (BuildContext context, int index) => Text('Item $index')`